src/css/labels.css

```diff
@@ -118,3 +118,8 @@
   background: var(--color-yellow-300);
   color: var(--color-black);
 }
+
+.label--role-admin {
```

Comment:
If we change this rule to match `.label--role-admin-only` you will get, by default, 'Admin Only' as the label text. That is what you want, bar the headline style caps.

Reply:
Got it! Thanks Neil. Please can you change this to `.label--role-admin-only` then? :)